NOTICE

```diff
@@ -0,0 +1,14 @@
+Second State Containerd shim of Wasmedge
```

Comment:
To be honest, I am not copyright / license expert. I add this file is because daace49773f473335f348030052146295670bc6a draw piece of code from Youki (APL 2.0) into runwasi. According the license I think we should expose more detail about we use from where.

Maybe the file is unnecessary or should be rewritten from repo's owner perspective. I will respect your decision.

Reply:
This really concerns me. It seems like https://github.com/deislabs/runwasi/pull/26/files#diff-8f8704e2bb5644be97cfbd99c736cb7f432da2de20292de2b0f69e4193bd0f64R1425-R1475 are copied from Youki verbatim https://github.com/containers/youki/blob/af45666b090a3c7e5ca7139061fcc2bfbd88c015/crates/libcontainer/src/rootfs/utils.rs#L69-L121. 

I'd raise the concern to my internal team for a review.